app/src/main/java/com/example/finding_tory/ItemViewActivity.java

```diff
@@ -12,13 +12,18 @@
 
 import com.google.android.material.floatingactionbutton.FloatingActionButton;
 
+import java.io.Serializable;
 import java.text.SimpleDateFormat;
 import java.util.Locale;
 
-public class ItemViewActivity extends AppCompatActivity {
+
+public class ItemViewActivity extends AppCompatActivity implements Serializable {
```

Comment:
I don't think the activity needs to be serializable.
```suggestion
public class ItemViewActivity extends AppCompatActivity {
```